benchmarks/000.microbenchmarks/020.network-benchmark/input.py

```
@@ -3,5 +3,5 @@
 def buckets_count():
     return (0, 1)
 
-def generate_input(data_dir, size, input_buckets, output_buckets, upload_func):
+def generate_input(data_dir, size, benchmarks_bucket, input_paths, output_paths, upload_func, nosql_func):
     return {'output-bucket': output_buckets[0]}
```

Ruff (0.8.2)

Undefined name `output_buckets`

(F821)